python/rubin/rag/extra_scripts/scrape_github.py

```diff
@@ -43,68 +52,102 @@ def load_files_1repo(repo: str = "lsst/daf_butler") -> list:
     docs : list
         list of langchain documents
     """
-    # note that it's necessary to have set the env var
-    # GITHUB_PERSONAL_ACCESS_TOKEN to the relevant GitHub API access token
-    access_token = str(os.getenv("GITHUB_PERSONAL_ACCESS_TOKEN"))
-
     loader = GithubFileLoader(
         repo=repo,
         branch="main",
         github_api_url="https://api.github.com",
-        access_token=access_token,
+        access_token=str(access_token),
         file_filter=None,
     )
 
     docs: list = []
 
-    for metadata in loader.get_file_paths():
+    try:
+        file_paths = loader.get_file_paths()
+    except Exception:
+        _log.exception("Repo {repo} may not have a branch named 'main'.")
+        return []
+
+    for metadata in file_paths:
         file_path = metadata["path"]
 
         try:
             string = loader.get_file_content_by_path(file_path)
             doc = Document(string, metadata=metadata)
             docs.append(doc)
         except Exception:
-            logging.exception("Failed to load file.")
+            _log.exception("Failed to load file.")
 
     return docs
 
 
-def load_org(n_repo_max: int = 2, org_name: str = "lsst-dm") -> list:
+def repos_in_org(org_name: str = "lsst-dm") -> list:
+    """Get list of repos with a GitHub organization.
```

Comment:
is this supposed to be within?
```suggestion
    """Get list of repos within a GitHub organization.
```

Reply:
Thanks, good catch, done in [a6cc243](https://github.com/lsst-dm/rubin_rag/pull/28/commits/a6cc243c00d5cfc85d6ec0098ea4a8f8350dc361)